### PR TITLE
Removed hard-coded .vqa assumptions from the engine

### LINKED ISSUE #12775: Move remaining Westwood file formats to Mods.Cnc
Just VQA left:

- [ ] `Scripting\Media.cs` calls `VqaReader` directly
- [ ] `Scripting\Global\MediaGlobal.cs` calls `VqaReader` directly
- [ ] `AssetBrowserLogic` needs `VqaPlayerWidget`, which also calls `VqaReader`
- [ ] `MissionBrowserLogic` needs `VqaPlayerWidget`, which also calls `VqaReader`

Note: `ShpTS` and `RLEZerosCompression` should remain in `Common` as discussed on IRC, as ShpTS has the best community tool support (and its `RLEZerosCompression` is non-proprietary).

### PR DESCRIPTION
**Pull request by Mailaender**
Closes #12775.

### CODE REVIEW

File: OpenRA.Mods.Common/Scripting/Media.cs [hunk 1]
```diff
@@ -74,13 +74,14 @@ public static void PlayFMVInRadar(World w, VqaReader movie, Action onComplete)
...
+			var player = Ui.Root.Get<VideoPlayerWidget>("PLAYER");
 			player.Stop();
 		}
 
-		public static VqaReader LoadVqa(Stream s)
+		public static IVideo LoadVideo(Stream s)
 		{
-			return new VqaReader(s);
+			var video = VideoLoader.GetVideo(s, Game.ModData.VideoLoaders);
+			return video;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Scripting/Media.cs]: `return VideoLoader.GetVideo(s, Game.ModData.VideoLoaders);`?

**Review by abcdefg30 — COMMENTED** [review]
Changes look good to me, but I didn't finish testing due to #18823.

File: OpenRA.Mods.Common/Scripting/Media.cs (hunk 1 above)
Comment (Mailaender) [reply OpenRA.Mods.Common/Scripting/Media.cs]: Yep.

**Review by abcdefg30 — DISMISSED** [review]

**Comment by reaperrr** [comment]
Looks good to me. RA and TD videos work (tested briefings and asset browser). Didn't test D2k.
The TS video I tested (MEKATAK2 from FS) didn't work, but I have my doubts whether we've ever updated our VqaReader to support those.

LGTM, needs a minor rebase, though.

**Review by reaperrr — DISMISSED** [review]

**Comment by Mailaender** [comment]
Rebased.

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/5979493359ffafbbad8732565e00a2b8195f7b90...7a756f3358f91dc6d99446a88b76c0616c886727)

**Pull request merged**